panel/src/components/Forms/Input/CheckboxesInput.vue

```diff
@@ -9,7 +8,6 @@
 			<k-choice-input v-bind="choice" @input="input(choice.value, $event)" />
 		</li>
 	</ul>
-	<k-empty v-else icon="info">{{ $t("options.none") }}</k-empty>
```

Comment:
The inputs shouldn't be responsible for this. We can handle that on a higher level in inputbox components.

Reply:
But should we do that change then only when the input box component is added?

Reply:
This has been added somewhere in the alphas. The inputs never dealt with empty options in v3. But I also don't mind to keep it in there until the inputboxes are ready